components/prism-editorconfig.js

```diff
@@ -2,15 +2,19 @@ Prism.languages.editorconfig = {
 	// https://editorconfig-specification.readthedocs.io/en/latest/
 	'comment': /[;#].*/,
 	'section': {
-		pattern: /^[ \t]*\[.+]/m,
+		pattern: /(^|[ \t]*)\[.+]/,
```

Comment:
I think that's not what you meant. Since the lookbehind can match the empty string in the `[ \t]*` alternative, the lookbehind will trivially accept any string. This means that it will even match things like this:

```
foo[what?]
```

```suggestion
		pattern: /(^[ \t]*)\[.+\]/m,
```

Reply:
Yes, you're right. Fixed it.